### PR TITLE
Backmerge rel/latest for SDK 11.11.0 release

### PR DESCRIPTION
**Pull request by tylerlevine**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -381,10 +349,10 @@
   dependencies:
     buffer "^5.4.3"
 
-"@bitgo/statics@^6.3.0-rc.1":
-  version "6.3.0-rc.1"
-  resolved "https://registry.yarnpkg.com/@bitgo/statics/-/statics-6.3.0-rc.1.tgz#b62b585c012836528d0fdf8156f0f51fe51ef50c"
-  integrity sha512-LkjkMSoUMXRjYQ0d5gqThNhLY7m2+8FRcE/lYJZ9QObT8kPcZpMNTkYVk2gE0eub4DldxI1UMvJ+8Y3Kny8bDg==
+"@bitgo/statics@^6.2.1":
+  version "6.2.1"
```
Comment (james-callahan) [inline yarn.lock:353]: Why is this getting downgraded?

Comment (tylerlevine) [reply yarn.lock:353]: It was incorrectly upgraded to an release candidate in a merged PR. We should only ever have stable versions of packages inside this monorepo in the package.jsons and yarn.lock

Comment (james-callahan) [reply yarn.lock:353]: I don't see that noted in e.g. a commit message?

Comment (tylerlevine) [reply yarn.lock:353]: It was changed during the merge conflict resolution here: https://github.com/BitGo/BitGoJS/pull/1071/commits/fffb9fbb030c815603b556b20cbc2cd6e2ff27c4

**Comment by tylerlevine** [comment]
Needs some fixing to resolve merge conflicts - going to close this PR so I can open a new one off of a different branch, since I don't want to merge `master` into `rel/latest`.

**Pull request closed without merging**